js/solstice.js

```diff
@@ -121,7 +121,12 @@
   });
 
   eclipseFdnVideos.replace();
-
+  
+  // Make the whole block-list clickable
+  $('.block-list').click(function() {
+    $link = $(this).find('a').first();
+    $link[0].click();
```

Comment:
I did not test this, but I assume this will trow an error if link is undefined.

Would it be possible to do .find('h2 a') to make sure that the code links to the heading link and not a link in the summary?

Reply:
I'll test this but I think you are right.

If we agree on going with H4 for the headings, I could do find('h4 a').

Reply:
+1